bundles/org.openhab.ui.basic/web-src/_layout.scss

```diff
@@ -217,6 +217,25 @@
 			box-shadow: none;
 			-webkit-box-shadow: none;
 		}
+		.mdl-textfield {
+			&__input,
+			&__label {
+				font-family: "Helvetica","Arial",sans-serif;
+				font-size: 14px;
+				text-align: right;
+			}
+			&__input {
+				font-weight: 700;
+			}
+			&__label {
```

Comment:
I am not at all familiar with CSS. Why do you create several "blocks" for &__input et for &__label, rather than defining all properties in one "block" for each of them ?

Reply:
Ok, thank you for the explanation.